source/includes/steps-create-data-encryption-key.yaml

```diff
@@ -0,0 +1,120 @@
+  :ref:`Create a Master Key <fle-create-a-master-key>` section:
+
+  .. tabs::
+
+     tabs:
+
+       - id: use-cases-fle-guide-java
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+              :emphasize-lines: 11
+
+              String path = "master-key.txt";
+
+              byte[] fileBytes = new byte[96];
+
+              try (FileInputStream fis = new FileInputStream(path)) {
+                fileBytes = fis.readAllBytes();
+              } catch (IOException e) {
+                e.printStackTrace();
+              }
+
+              final byte[] localMasterKey = Arrays.copyOf(fileBytes, 96);
+
+
+---
+title: Specify KMS provider settings
+ref: specify-kms-provider-settings
+level: 4
+content: |
+  Next, specify the KMS provider settings. Set the provider name to ``local``
+  when using a local master key:
+
+  .. tabs::
+
+     tabs:
+
+       - id: use-cases-fle-guide-java
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+              :emphasize-lines: 4,5
+
+              String kmsProvider = "local";
+
+              Map<String, Map<String, Object>> kmsProviders = new HashMap<String, Map<String, Object>>() {{
+                put(kmsProvider, new HashMap<String, Object>() {{
+                  put("key", localMasterKey);
+                }});
+              }};
+
+
+---
+title: Construct the client
+ref: construct-the-client
+level: 4
+content: |
+  Create the client using the MongoDB connection string and key vault 
+  namespace. In this example, we specify the ``encryption`` database and the 
+  ``__keyVault`` collection:
+
+  .. tabs::
+
+     tabs:
+
+       - id: use-cases-fle-guide-java
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+              :emphasize-lines: 8,9,12
+
+              String connectionString = "mongodb://localhost:27017";
+              String keyVaultNamespace = "encryption.__keyVault";
+
+              ClientEncryptionSettings clientEncryptionSettings = ClientEncryptionSettings.builder()
+                .keyVaultMongoClientSettings(MongoClientSettings.builder()
+                  .applyConnectionString(new ConnectionString(connectionString))
+                  .build())
+                .keyVaultNamespace(keyVaultNamespace)
+                .kmsProviders(kmsProviders)
+                .build();
+
+              ClientEncryption clientEncryption = ClientEncryptions.create(clientEncryptionSettings);
+
+
+  .. note::
+
+     Ensure that the client has `ReadWrite
+     <https://docs.mongodb.com/manual/reference/built-in-roles/#readWrite>`_
+     permissions on the specified key vault namespace.
+
+
+---
+title: Verify the data encryption key was created
+ref: verify-data-key-created
+level: 4
+content: |
+  Verify that the key was created in your MongoDB replica set in the specified
+  key vault namespace. The document should resemble the following:
```

Comment:
It'd be nice to have a quick explanation of what the document actually is/contains/represents beyond just the example

Reply:
I removed that description from the beginning of the section because I did not think it provided very useful information. Which of the fields listed here https://github.com/mongodb/docs-ecosystem/pull/535/files#diff-54590a1d9915ddb3aea9c0163a7a6c8cR111 would you consider important? Though I did not directly refer to the names, I described the document contains _id, masterKey, and keyMaterial fields.
Should I also mention keyAltNames (which I have not attempted to generate)?

Reply:
I don't think we necessarily need to mention specific fields. Maybe just "includes the key (or KMS info) and metadata that describes when the key was created and last updated".

Reply:
Great, let me know if this makes sense:

https://github.com/mongodb/docs-ecosystem/pull/535/commits/58163994ae3c0dc20ee19ba73fbfe17a960b90db